README.md

```diff
@@ -468,21 +513,18 @@ output: "syft-table"
 # same as -q ; SYFT_QUIET env var
 quiet: false
```

Comment:
Is the yaml section of the README generated or programmatically validated? If not, should it be?

Reply:
when we started doing this we didn't have a way to generate this. Today we do, but we don't have any automation to patch the README.md in the right section with the correct information. (down for adding that, but it should be it's own issue/PR)